python-avd/pyavd/_eos_designs/structured_config/network_services/port_channel_interfaces.py

```diff
@@ -20,122 +23,93 @@ class PortChannelInterfacesMixin(Protocol):
     Class should only be used as Mixin to a AvdStructuredConfig class.
     """
 
-    @cached_property
-    def port_channel_interfaces(self: AvdStructuredConfigNetworkServicesProtocol) -> list | None:
+    @structured_config_contributor
+    def port_channel_interfaces(self: AvdStructuredConfigNetworkServicesProtocol) -> None:
         """
-        Return structured config for port_channel_interfaces.
+        Set structured config for port_channel_interfaces.
 
         Only used with L1 network services
         """
         if not self.shared_utils.network_services_l1:
-            return None
+            return
 
-        # Using temp variables to keep the order of interfaces from Jinja
-        port_channel_interfaces = []
-        subif_parent_interfaces = []
+        subif_parent_interfaces: list[EosCliConfigGen.PortChannelInterfacesItem] = []
```

Comment:
change this to `EosCliConfigGen.PortChannelInterfaces` to detect two auto generated parents with conflicting information. Which would break also today.

Also add a test where the parent is NOT autogenerated and make sure it still works.